Resources/Locale/en-US/preferences/ui/markings-picker.ftl

```diff
@@ -14,7 +14,7 @@ marking-slot-remove = Remove
 # Categories
 
 markings-category-Hair = Hair
-markings-category-FacialHair = Facial Hair
+markings-category-FacialHair = Front
```

Comment:
Could you leave a comment noting that this is a CD change?

Reply:
I could've sworn I reverted this